docs/iris/src/whatsnew/contributions_1.9/deprecate_2015-11-01_unit-module.txt

```diff
@@ -0,0 +1 @@
+* deprecated :mod:`iris.unit`
```

Comment:
You may want to mention the replacement module here too, in case the casual what's new browser gets alarmed that the units module is being removed with no mention of replacement.